sdks/go/pkg/beam/io/textio/testfile.txt

```diff
@@ -0,0 +1 @@
ooooasdfasdfasdfasdfasdfdsfdffooooasdfasdfasdfasdfasdfdsfdffooooasdfasdfasdfasdfasdfdsfdffooooasdfasdfasdfasdfasdfdsfdffooooasdfasdfasdfasdfasdfdsfdffooooasdfasdfasdfasdfasdfdsfdffooooasdfasdfasdfasdfasdfdsfdffooooasdfasdfasdfasdfasdfdsfdffooooasdfasdfasdfasdfasdfdsfdffooooasdfasdfasdfasdfasdfdsfdffooooasdfasdfasdfasdfasdfdsfdffooooasdfasdfasdfasdfasdfdsfdffooooasdfasdfasdfasdfasdfdsfdffooooasdfasdfasdfasdfasdfdsfdffooooasdfasdfasdfasdfasdfdsfdffooooasdfasdfasdfasdfasdfdsfdffooooasdfasdfasdfasdfasdfdsfdffooooasdfasdfasdfasdfasdfdsfdffooooasdfasdfasdfasdfasdfdsfdffooooasdfasdfasdfasdfasdfdsfdffooooasdfasdfasdfasdfasdfdsfdffooooasdfasdfasdfasdfasdfdsfdffooooasdfasdfasdfasdfasdfdsfdffooooasdfasdfasdfasdfasdfdsfdffooooasdfasdfasdfasdfasdfdsfdffooooasdfasdfasdfasdfasdfdsfdffooooasdfasdfasdfasdfasdfdsfdffooooasdfasdfasdfasdfasdfdsfdffooooasdfasdfasdfasdfasdfdsfdffooooasdfasdfasdfasdfasdfdsfdffooooasdfasdfasdfasdfasdfdsfdffooooasdfasdfasdfasdfasdfdsfdffooooasdfasdfasdfasdfasdfdsfdffooooasdfasdfasdfasdfasdfdsfdffooooasdfasdfasdfasdfasdfdsfdffooooasdfasdfasdfasdfasdfdsfdffooooasdfasdfasdfasdfasdfdsfdffooooasdfasdfasdfasdfasdfdsfdffooooasdfasdfasdfasdfasdfdsfdffooooasdfasdfasdfasdfasdfdsfdffooooasdfasdfasdfasdfasdfdsfdfooooasdfasdfasdfasdfasdfdsfdfffooooasdfasdfasdfasdfasdfdsfdffooooasdfasdfasdfasdfasdfdsfdffooooasdfasdfasdfasdfasdfdsfdffooooasdfasdfasdfasdfasdfdsfdffooooasdfasdfasdfasdfasdfdsfdffooooasdfasdfasdfasdfasdfdsfdffooooasdfasdfasdfasdfasdfdsfdffooooasdfasdfasdfasdfasdfdsfdffooooasdfasdfasdfasdfasdfdsfdffooooasdfasdfasdfasdfasdfdsfdffooooasdfasdfasdfasdfasdfdsfdffooooasdfasdfasdfasdfasdfdsfdffooooasdfasdfasdfasdfasdfdsfdffooooasdfasdfasdfasdfasdfdsfdffooooasdfasdfasdfasdfasdfdsfdffooooasdfasdfasdfasdfasdfdsfdffooooasdfasdfasdfasdfasdfdsfdffooooasdfasdfasdfasdfasdfdsfdffooooasdfasdfasdfasdfasdfdsfdffooooasdfasdfasdfasdfasdfdsfdffooooasdfasdfasdfasdfasdfdsfdffooooasdfasdfasdfasdfasdfdsfdffooooasdfasdfasdfasdfasdfdsfdffooooasdfasdfasdfasdfasdfdsfdffooooasdfasdfasdfasdfasdfdsfdffooooasdfasdfasdfasdfasdfdsfdffooooasdfasdfasdfasdfasdfdsfdffooooasdfasdfasdfasdfasdfdsfdffooooasdfasdfasdfasdfasdfdsfdffooooasdfasdfasdfasdfasdfdsfdffooooasdfasdfasdfasdfasdfdsfdffooooasdfasdfasdfasdfasdfdsfdffooooasdfasdfasdfasdfasdfdsfdffooooasdfasdfasdfasdfasdfdsfdffooooasdfasdfasdfasdfasdfdsfdffooooasdfasdfasdfasdfasdfdsfdffooooasdfasdfasdfasdfasdfdsfdffooooasdfasdfasdfasdfasdfdsfdffooooasdfasdfasdfasdfasdfdsfdffooooasdfasdfasdfasdfasdfdsfdffooooasdfasdfasdfasdfasdfdsfdffooooasdfasdfasdfasdfasdfdsfdffooooasdfasdfasdfasdfasdfdsfdffooooasdfasdfasdfasdfasdfdsfdffooooasdfasdfasdfasdfasdfdsfdffooooasdfasdfasdfasdfasdfdsfdffooooasdfasdfasdfasdfasdfdsfdffooooasdfasdfasdfasdfasdfdsfdffooooasdfasdfasdfasdfasdfdsfdffooooasdfasdfasdfasdfasdfdsfdffooooasdfasdfasdfasdfasdfdsfdffooooasdfasdfasdfasdfasdfdsfdffooooasdfasdfasdfasdfasdfdsfdffooooasdfasdfasdfasdfasdfdsfdffooooasdfasdfasdfasdfasdfdsfdffooooasdfasdfasdfasdfasdfdsfdffooooasdfasdfasdfasdfasdfdsfdfooooasdfasdfasdfasdfasdfdsfdfff
```

Comment:
We have a license check (the RAT precommit), which requires that all files have the Apache licence header, which this file is triggering.

To have the check ignore this new test data file, please add 
    "**/go/**/*.txt",
to
https://github.com/apache/beam/blob/master/build.gradle#L81

Or move this test data to .../sdks/go/data/...

Reply:
done. moved to `go/data/..`